incubator/cloudbeaver/values.yaml

```diff
@@ -0,0 +1,121 @@
+init:
+  #This image is used by the 2nd init container which relies on the jar binary to inject the xml files
+  image:
+    registry: docker.io
+    repository: openjdk
+    tag: 11-jdk
+    pullPolicy: IfNotPresent
+
+cloudbeaver:
+  #Cloudbeaver image details
+  image:
+    registry: docker.io
+    repository: dbeaver/cloudbeaver
+    tag: latest
```

Comment:
I think that we should pin the version of CloudBeaver that we tested with. If you pin the version in `Chart.yaml`, then this becomes `"{{ .Chart.AppVersion }}"`.

Reply:
Done!